scss/config/_scale.scss

```diff
@@ -56,7 +56,7 @@ $text-sizes: (
 /// A set of standard spacing sizes.
 ///
 /// @group config-scale
-/// @preview sizes; style: ruler
+/// @sizes style: ruler
```

Comment:
Is there a reason we decided to keep using keyword args for style & key, rather than following the syntax used elsewhere?

Reply:
No, the only reason is that I made a minimal change, as at that point I did not yet know how you wanted it to be.

Reply:
@mirisuzanne What syntax used elsewhere are you referring to? How would you like this to work?

Reply:
I left comments on the slack thread yesterday. Icons and fonts both take a bare key string as the first argument, and fonts take an additional parenthetical arg. That matches more closely with other SassDoc annotations.

For colors/ratios (which only take a key), that might look like:

```scss
/// @colors brand-colors
/// @ratios text-ratios
```

For sizes (which take both key and style), maybe: 

```scss
/// @sizes <key> (<style>)
/// @sizes spacing-sizes (ruler)
```

Reply:
Hrm, that's the problem with Slack threads -- no one sees them except the author.

Some thoughts:

1) Right now, `@font` expects the key to be within `'` or `"`, but `@icons` does not. Is there any reason for the difference? We should probably standardize, maybe just doing away with quotes entirely.
2) `@ratios` also accepts a `count` arg, not just a `key`.
3) I generally like the syntax you're proposing, but the downside is that all of the arguments are implicit -- you have to know what arguments are expected, in what order. In some ways that seems like a step backward from `key:value` args. For instance, if we expect `@sizes (ruler)` and `@ratios (4)`, it's not clear (without looking at the API documentation) what those arguments mean. But you're correct that it matches SassDoc's API more closely, although I don't see any SassDoc annotations using `(...)` for arguments.

Reply:
1. SassDoc doesn't use `'` or `"`, so I vote to drop it.
2. 👍 good catch.
3. 
  a. Yeah, I think it would be great if both were available. Is that an option?
  b. It looks like SassDoc uses `{}` for types, `[]` for defaults, and `<>` for links. They sometimes use nothing if they expect e.g. a link first and then optional text. The `style` arg might make sense as a type: `@sizes {ruler} spacing-sizes` (ideally the order doesn't matter). Ratio-count makes less sense there, but I don't like the way our ratio-previews work anyway, so maybe I should just do that redesign, and try to eliminate that argument.

Reply:
- `1`: Ok, I'll ditch the quotes for `@font`.
- `3a`: Hm, I'm not sure the benefit of supporting two different APIs is worth the added complexity?

SassDoc generally does not allow "any order" for the annotation arguments. I'd propose we follow their lead for these, such that the first word (anything except whitespace) is interpreted as the `key`, and then the args follow. I'll leave `@font` as-is with `()` for variants and `{}` for formats. For `@sizes`, I'll go with `{}` for the style. @mirisuzanne That sound okay?